docs/howto/operator.rst

```diff
@@ -612,25 +615,113 @@ Using the operator
 .. literalinclude:: ../../airflow/contrib/example_dags/example_gcp_spanner.py
     :language: python
     :dedent: 4
-    :start-after: [START howto_operator_spanner_deploy]
-    :end-before: [END howto_operator_spanner_deploy]
+    :start-after: [START howto_operator_spanner_database_deploy]
+    :end-before: [END howto_operator_spanner_database_deploy]
 
 Templating
 """"""""""
 
 .. literalinclude:: ../../airflow/contrib/operators/gcp_spanner_operator.py
-  :language: python
-  :dedent: 4
-  :start-after: [START gcp_spanner_deploy_template_fields]
-  :end-before: [END gcp_spanner_deploy_template_fields]
+    :language: python
+    :dedent: 4
+    :start-after: [START gcp_spanner_database_deploy_template_fields]
+    :end-before: [END gcp_spanner_database_deploy_template_fields]
 
 More information
 """"""""""""""""
 
-See Google Cloud Spanner API documentation for instance `create
-<https://cloud.google.com/spanner/docs/reference/rpc/google.spanner.admin.instance.v1#google.spanner.admin.instance.v1.InstanceAdmin.CreateInstance>`_
-and `update
-<https://cloud.google.com/spanner/docs/reference/rpc/google.spanner.admin.instance.v1#google.spanner.admin.instance.v1.InstanceAdmin.UpdateInstance>`_.
+See Google Cloud Spanner API documentation for `database create
+<https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances.databases/create>`_
+
+CloudSpannerInstanceDatabaseUpdateOperator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Runs a DDL query in a Cloud Spanner database. It allows you to modify structure of an
+existing database.
+
+The DDL queries run through the operator should be done in an idempotent way if only
```

Comment:
Rephrase: `The DDL queries should be run through the operator in an idempotent way`

Reply:
I rephrased it slightly differently.